src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig

```diff
@@ -144,7 +144,12 @@
 
                     <div class="hidden" id="sf-trace-{{ discr }}-{{ loop.index0 }}">
                         <div class="trace">
-                            {{ caller.file|file_excerpt(caller.line) }}
+                            {{ caller.file|file_excerpt(caller.line)|replace({
```

Comment:
The replace must be done in the `CodeExtension` extension in the TwigBridge, it might not be related to this PR, right?
I have not understood your 2nd statement

Reply:
this is done so the selector `body.theme-dark .trace span[style="color: #222222"] ` works, isnt it? which on itself overwrites the color again.

So why not use the old color here, i.e. `span[style="color: #0000BB"]`

Reply:
you could replace the colors with `var(--some-color)` instead of a different hardcoded color.

Reply:
And no, CodeExtension should not perform color-replacements which are specific to the profiler themes.

Reply:
The old colors don't display correctly on dark theme. I did this change to match what we have on Exception page, and do not have to write multiple css statements to rewrite colors.